CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contribution Guide
+
+Firstly, thank you for considering making a contribution to our project! It's people like you that make Operation Code such a great community.
+
+We're an open source project and we love receiving contributions from our community - you! There are many ways to contribute to our projects, from writing tutorials or blog posts, improving documentation, submitting bug reports or feature requests, or writing code which can be merged into any of our repositories.
+
+The team at Operation Code urges all contributors to join our Slack team. Participating in discussions with the community on our Slack channel, is the best way to run new ideas by the team, and is the best place to get help. You can get an invitation to our Slack channel by [requestion to join Operation Code](https://operationcode.org/join). Once in our Slack team, simply type: '/open #oc-projects' and then click enter. Feel free to ask for help; everyone is a beginner at first :smile_cat:!
```

Comment:
The channel associated with this particular project is actually #oc-python-projects. Can you please change this to read `Once in our Slack team, simply type: '/open #oc-python-projects'`?